recipes/mozjpeg/all/conanfile.py

```diff
@@ -129,22 +136,27 @@ def build(self):
 
     def package(self):
         self.copy(pattern="LICENSE.md", dst="licenses", src=self._source_subfolder)
-        if self.settings.os == "Windows":
+        if self._use_cmake:
             cmake = self._configure_cmake()
             cmake.install()
             tools.rmdir(os.path.join(self.package_folder, "doc"))
         else:
             autotools = self._configure_autotools()
             autotools.install()
-            tools.rmdir(os.path.join(self.package_folder, "share"))
-            tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-            for la_file in glob.glob(os.path.join(self.package_folder, "lib", "*.la")):
-                os.remove(la_file)
+            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
```

Comment:
Please require min conan version for this new feature

Reply:
Done